### DISCUSSION TITLE
Seeed-LoRa-E5.ioc - Link Layer Specificaiton Bug

### DISCUSSION THREAD

**Discussion by LucasAMPA** [danak6jq/Seeed-LoRa-E5#25] [Ideas]
Using the .ioc I believe I found a small bug. Here is my setup:

STM32CubeIDE FW 12.0.0
CubeMX: 6.8.0

Inside of the .ioc on Select the LoRaWAN Link Layer specification I chose the option v.1.0.4
Searching all the files in Visual Studio I see there are about 8 files referencing the "CONTEXT_MANAGEMENT_ENABLED". 
In lorawan_conf.h the value was correctly changed to 1 after enabling it in the .ioc file
In the Middlewares/LoRaWAN/LmHandler/NvmDataMgmt.c file, CONTEXT_MANAGEMENT_ENABLED still remains 0

**Comment by danak6jq** [comment]
Please refer to the latest FW1.4.0 branch (now the repo default branch) for correct use of Context Management.
